### ISSUE TITLE
viewer feature request

### ISSUE THREAD

**Issue by tsentered** [manga-download/hakuneko#2471]
The current Hakuneko viewer for Mac does not respond to the finger gestures for expanding and contracting size of images or the double finger tap (right click equivalent for windows.)

This was an issue visited previously. Will this feature be added in future versions? Besides that Haku has the best reader hands down.

Thanks either way.

Regards

**Comment by 27136773** [comment]
I think no contributor is going to pick this up, however i will leave this open in case of HakuNeko NW will adapt this...
The image viewer was originally just meant to be a preview for checking image quality of the website or artwork of the manga.

**Comment by tsentered** [comment]
Yeah I kind of figured it would be a huge pain to undertake such a thing. Thanks anyways.
